src/utils/style.css

```
@@ -24,10 +24,18 @@ a:hover {
 }
 
 body {
+  color: #333;
+  font-family:
+    "Helvetica Neue", "Hiragino Sans", "Hiragino Kaku Gothic ProN", "Meiryo",
```

Stylelint (17.12.0)

[error] Expected no quotes around "Meiryo" (font-family-name-quotes)

(font-family-name-quotes)